paasta_tools/cli/schemas/tron_schema.json

```diff
@@ -0,0 +1,136 @@
+  "description": "tron on paasta yaml (docs todo)",
+  "type": "object",
+  "definitions": {
+    "name": {
+        "type": "string",
+        "pattern": "^[A-Za-z_][\\w\\-]{0,254}$"
+    },
+    "time_delta": {
+        "type": "string",
+        "pattern": "^\\d+\\s*[a-z]+$"
+    },
+    "action": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["command"],
+      "properties": {
+        "name": {"$ref": "#definitions/name"},
+        "command": {"type": "string"},
+        "node": {"$ref": "#definitions/name"},
+        "requires": {
+          "type": "array",
+          "items": {"type": "string"}
+        },
+        "retries": {
+          "type": "integer",
+          "minimum": 0,
+          "exclusiveMinimum": false
+        },
+        "executor": {"enum": ["ssh", "paasta"]},
+        "cpus": {
+          "type": "number",
+          "minumum": 0,
+          "exclusiveMinimum": true
+        },
+        "mem": {
+          "type": "number",
+          "minumum": 32,
+          "exclusiveMinimum": true
+        },
+        "constraints": {
+          "type": "array",
+          "items": {"type": "array"},
+          "uniqueItems": true
+        },
+        "service": {"type": "string"},
+        "deploy_group": {"type": "string"},
+        "pool": {"type": "string"},
+        "env": {
+          "type": "object",
+          "additionalProperties": {"type": "string"}
+        },
+        "extra_volumes": {
+          "type": "array",
+          "items": {"type": "object"},
+          "uniqueItems": true
+        },
+        "cluster": {"type": "string"},
+        "expected_runtime": {"$ref": "#definitions/time_delta"}
+      }
+    }
+  },
+  "required": ["jobs"],
+  "properties": {
+    "jobs": {
+      "type": ["null", "array"],
+      "items": {
+        "type": "object",
+        "required": ["name", "node", "schedule", "actions"],
+        "additionalProperties": false,
+        "properties": {
+          "name": {"$ref": "#definitions/name"},
+          "node": {"$ref": "#definitions/name"},
+          "schedule": {
+            "type": ["string", "object"]
+          },
+          "actions": {
+            "type": "array",
+            "items": {"$ref": "#definitions/action"}
+          },
+          "monitoring": {
+            "type": "object",
+            "properties": {
+                "team": {"type": "string"},
+                "runbook": {"type": "string"},
+                "page": {"type": "boolean"},
+                "tip": {"type": "string"},
+                "notification_email": {"type": "string"},
+                "realert_every": {
+                    "type": "integer",
+                    "minimum": 1,
+                    "exclusiveMinimum": false
+                },
+                "dependencies": {
+                    "type": "array",
+                    "items": {"type": "string"}
+                },
+                "irc_channels": {
```

Comment:
Can you add slack_channels while you are at it? I guess I need to add that to our other schemas

Reply:
Sure, done.